osmaxx-py/osmaxx/excerptexport/models/output_file.py

```diff
@@ -19,6 +22,17 @@ class OutputFile(models.Model):
     extraction_order = models.ForeignKey(ExtractionOrder, related_name='output_files',
                                          verbose_name=_('extraction order'))
 
+    @property
+    def download_file_name(self):
+        return excerptexport_settings.APPLICATION_SETTINGS['download_file_name'] % {
```

Comment:
This is even more confusing than the name `settings`: APPLICATION_SETTINGS should really come from the the application, not from a settings file somewhere in the wild.

Furthermore, the return looks more like a call to a template with all the context variables, can we change it to something more understandable (what is it doing right now here?).